qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -104,8 +104,12 @@
             gagent_guest_reboot_pattern = "Restarting system"
             Windows:
                 gagent_guest_reboot_pattern = ""
+            s390x:
+                gagent_guest_reboot_pattern = "System Reboot"
         - check_halt:
             gagent_check_type = halt
+            s390x:
+                gagent_guest_shutdown_pattern = "System Halt"
```

Comment:
@BohdanMar Bogdan, I missed one thing, the pattern specified in lines 111/112 would be overwritten by line 114, so please move it after that line.